tools/docker/README.md

```diff
@@ -18,38 +18,33 @@ _**All commands mentioned in this document should be run from the base Jetpack d
 
 ### Prerequisites
 
-* [Docker](https://hub.docker.com/search/?type=edition&offering=community)
-* [NodeJS](https://nodejs.org)
-* [Pnpm](https://pnpm.io/)
-* Optionally [Ngrok](https://ngrok.com) client and account or some other service for creating a local HTTP tunnel. It’s fine to stay on the free pricing tier with Ngrok.
+To use Jetpack's Docker environment, you will need:
 
-Install prerequisites; you will need to open up Docker to install its dependencies.
+- A [local copy of the Jetpack repository](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md##clone-the-repository).
+- [Docker](https://hub.docker.com/search/?type=edition&offering=community) installed and running.
+- [Jetpack's required tools](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#installing-development-tools).
 
-Start by cloning the Jetpack repository:
+Our Docker instance comes with a default settings file. You can modify those defaults by copying the file:
 
-```sh
-git clone git@github.com:Automattic/jetpack.git && cd jetpack
-```
-
-Optionally, copy settings file to modify it:
 ```sh
 cp tools/docker/default.env tools/docker/.env
 ```
 
 Anything you put in `.env` overrides values in `default.env`. You should modify all the password fields for security, for example.
 
-Finally, spin up the containers:
+Once you're all set with the above, spin up the containers:
```

Comment:
Once the reader gets here they're no longer working on pre-requisites but on starting docker. Perhaps introduce a new subheading here so it stands out, or keep this pre-reqs + starting under the same heading as now and reword it from 'Prerequisites' to 'Quick start', or similar?

Reply:
👍 Done in 6fe0551e8cdafba95246950a2ab7c0bf637d23c9